README.md

```diff
@@ -302,6 +292,14 @@ The software will recognize all previous labeling decisions, and a new model can
 | Software |     Q1      |
 |:--------:|:-----------:|
 
+# Excluded Software
+
+This section contains a list of software that did not fullfill the [inclusion criteria](#inclusion-criteria), but that are still largely used in the scientific community.
+
+## [RobotAnalyst](http://www.nactem.ac.uk/robotanalyst/)
+
+RobotAnalyst was developed as part of the Supporting Evidence-based Public Health Interventions using Text Mining project to support the literature screening phase of systematic reviews. The current version of RobotAnalyst is mounted on a University of Manchester server and is a prototype demo system for research purposes at Manchester University and partners.
+
```

Comment:
Maybe you could add a link to https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/issues/29#issuecomment-2517211394